validation/1.4/modules/ROOT/pages/validation-examples.adoc

```diff
@@ -0,0 +1,583 @@
+					<validation:ip value="193.1" />
+				</validation:ips>
+			</validation:white-list>
+		</validation:is-whitelisted-ip>
+	</flow>
+----
+
+== Configure the Matches Regex Operation
+
+The *Matches regex* operation validates that a given expression matches a Java regular expression.
+
+The following example shows how to configure the operation in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Validation > Matches regex*.
+. Drag *Matches regex* to the Studio canvas.
+. In the *Matches regex* configuration screen, for the *Expression* field select `Expression`, and in the empty field add  `#[vars.doReply]`.
+
+.Matches regex Configuration
+image::validation-matchesregex.png[Matches regex Configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<validation:matches-regex value="#[payload]" regex=".+\.log" caseSensitive="false"/>
+----
+
+== Configure the Validate Size Operation
+
+The *Validate size* operation validates that the input size is between the given minimum and maximum boundaries.
+This is valid for inputs of type String, Collection, Map, and Array. In the case of a String, the size refers to the length in characters.
+
+The following example shows how to configure the operation in Anypoint Studio:
+
+. In the *Mule Palette* view, select *Validation > Validate size*.
+. Drag *Validate size* to the Studio canvas.
+. In the *Validate size* configuration screen, complete the following parameters:
+
+* *Value* +
+`payload`
+* *Min* +
+`vars.minLength`
+* *Max* +
+`vars.maxLength`
+
+.Validate size Configuration
+image::validation-validatesize.png[Validate size Configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<validation:validate-size value="#[payload]" min="#[vars.minLength]" max="#[vars.maxLength]"/>
+----
+
+== Configure the All Scope
+
+The *All* scope evaluates a list of nested validation operations and informs only one `VALIDATION:MULTIPLE` error which summarizes all of the found errors.
+There are scenarios in which you may want to evaluate several conditions, out of which more than one could fail simultaneously. In these cases, it’s ideal to generate a single error that contains all of the descriptions. About the *All* scope validator:
+
+* All validations are executed even if all of them fail.
+
+* If any of the validations fail, one single exception is thrown. The exception contains a multiline message in which each line corresponds to every failing validation.
+
+* Validators are executed sequentially using the flow’s thread, but the order shouldn't matter because validators don’t cause any side effects.
+
+* Unlike the rest of the validation operations, the *All* scope validator does not enable you to customize the exception type or the message through `validation:exception` or exception factory elements. However, you can customize the message of the inner validation operations.
```

Comment:
This sounds like something that should come at the beginning, when we explain WHAT the scope vs. the operations are?
See proposed order of info above.